tests/src/test/scala/com/nvidia/spark/rapids/StringFunctionSuite.scala

```diff
@@ -21,7 +21,7 @@ import org.scalatest.funsuite.AnyFunSuite
 
 import org.apache.spark.SparkConf
 import org.apache.spark.sql.{DataFrame, Row, SparkSession}
-import org.apache.spark.sql.functions._
+import org.apache.spark.sql.functions.{col, lower, upper}
```

Comment:
Spark 3.5.0 introduced a new `printf` function, and importing that causes a compilation issue because we call Scala's `printf` in this code.